package.json

```diff
@@ -87,7 +87,7 @@
     "jsdoc": "3.6.3",
     "jsdoc-babel": "0.5.0",
     "lint-staged": "10.2.3",
-    "mongodb-runner": "4.8.0",
+    "mongodb-runner": "mongodb-js/runner",
```

Comment:
I just wanted to hear from @dplewis about this change. Otherwise we are good to go.